package.json

```diff
@@ -79,7 +79,7 @@
   },
   "homepage": "https://github.com/elastic/apm-agent-nodejs",
   "dependencies": {
-    "@elastic/ecs-pino-format": "^1.1.2",
+    "@elastic/ecs-pino-format": "^1.2.0",
```

Comment:
just noting the bumped dependency in cast it wasn't intentional.

Reply:
It was intentional to get [this change](https://github.com/elastic/ecs-logging-nodejs/blob/master/loggers/pino/CHANGELOG.md#v120) to support testing log correlation, as commented here:
https://github.com/elastic/apm-agent-nodejs/blob/3341137cdf70f925dac2541260b46f4ce4208333/test/disable-send.test.js#L84-L87